Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs

```diff
@@ -18,6 +19,24 @@ public class BaseDataGridColumn<TItem> : BaseDataGridComponent
     /// <returns>Formatted display value.</returns>
     public string FormatDisplayValue( object value )
```

Comment:
Should we make `FormatDisplayValue` virtual and override the logic in DataGridSelectColumn?